test/jdk/java/awt/Mixing/AWT_Mixing/OverlappingTestBase.java

```diff
@@ -454,36 +475,38 @@ protected void cleanup() {
      */
     protected Component currentAwtControl;
 
-    private void testComponent(Component component) throws InterruptedException, InvocationTargetException {
+    private void testComponent(Component component) throws InterruptedException,
+                                                           InvocationTargetException {
         Robot robot = null;
         try {
             robot = new Robot();
-        }catch(Exception ignorex) {
+        } catch(Exception ignored) {
         }
         currentAwtControl = component;
         System.out.println("Testing " + currentAwtControl.getClass().getSimpleName());
-        SwingUtilities.invokeAndWait(new Runnable() {
-            public void run() {
-                prepareControls();
-            }
-        });
+
+        SwingUtilities.invokeAndWait(() -> prepareControls());
+
         if (component != null) {
             Util.waitTillShown(component);
         }
         Util.waitForIdle(robot);
+
+        // wait for graphic effects on systems like Win7
+        Toolkit.getDefaultToolkit().sync();
         try {
-            Thread.sleep(500); // wait for graphic effects on systems like Win7
-        } catch (InterruptedException ex) {
+            Thread.sleep(500);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
         }
```

Comment:
You have `robot`, so you can use `robot.delay(500)` to avoid catching the exception.